packages/askui-nodejs/src/utils/transformations.ts

```diff
@@ -8,8 +8,8 @@ export async function toBase64Image(imagePath: string): Promise<string> {
   if (!(fs.existsSync(imagePath))) {
     throw new Error(`the image ${imagePath} does not exists!`);
   }
-  const image = await read(imagePath);
-  return image.getBase64Async('image/png');
+  const image = sharp(imagePath);
+  return `data:image/png;base64,${(await image.toBuffer()).toString('base64')}`;
 }
 
 export async function toBase64ImageIfNeeded(pngPathOrBase64Image: string): Promise<string> {
```

Comment:
I think we can and should extract some functions as this is quite a long function including multiple steps.